public/app/routes/routes.tsx

```diff
@@ -280,14 +279,18 @@ export function getAppRoutes(): RouteDescriptor[] {
     },
     {
       path: '/verify',
-      component: SafeDynamicImport(
-        () => import(/* webpackChunkName "VerifyEmailPage"*/ 'app/core/components/Signup/VerifyEmailPage')
-      ),
+      component: !config.verifyEmailEnabled
+        ? () => <Redirect to="/signup" />
+        : SafeDynamicImport(
+            () => import(/* webpackChunkName "VerifyEmailPage"*/ 'app/core/components/Signup/VerifyEmailPage')
+          ),
       pageClass: 'login-page sidemenu-hidden',
     },
     {
       path: '/signup',
-      component: SignupPage,
+      component: config.disableUserSignUp
+        ? () => <Redirect to="/login" />
```

Comment:
@tskarhed, I added `login` as a redirect URL because, `/` path is working, but throws an `unauthorized` error in the home dashboard.